manticore/core/smtlib/solver.py

```diff
@@ -411,7 +416,7 @@ def _pop(self):
         """Recall the last pushed constraint store and state."""
         self._send("(pop 1)")
 
-    def can_be_true(self, constraints: ConstraintSet, expression=True):
+    def can_be_true(self, constraints: ConstraintSet, expression: Union[bool, Bool] = True):
```

Comment:
This is missing a return statement
```suggestion
    def can_be_true(self, constraints: ConstraintSet, expression: Union[bool, Bool] = True) -> bool:
```

Reply:
Thanks; that looks good!  Without a return type annotation, `mypy` assumes `Any`, so indeed, the `bool` return type here is more specific (and matches the superclass).